### ISSUE TITLE
A fix for when running author list falls outside margins for classoption: "jou"

### ISSUE THREAD

**Issue by pederisager** [crsh/papaja#406]
I encountered the following bug/problem when listing multiple (12) co-authors and knitting the PDF using classoption "jou":

The automatically generated running author list trails off of the page, like so: 
![image](https://user-images.githubusercontent.com/20044958/91745540-67234d80-ebbb-11ea-98e0-6acb0025de1f.png)

I could not find an option for editing the running author list in the papaja documentation. However, I did find a working solution. Posting here in case of interest to anyone.

The fancyhdr package apparently lets you override the default running head and author list generated by "jou". To customize the header, add `header-includes :` to the end of the papaja rmarkdown header, and customize the `\lhead` and `\rhead` options: 
```
header-includes   :
- \usepackage{fancyhdr}
- \pagestyle{fancy}
- \lhead{AUTHOR NUMBER ONE ET AL.}
- \rhead{A VERY IMPORTANT PAPER}
```
Which yields: 
![image](https://user-images.githubusercontent.com/20044958/91745682-a94c8f00-ebbb-11ea-9c1a-302db1d535c1.png)

This does add some extra formatting into the mix however. It would be nice if I could simply edit the running authorlist in the same way I can edit the `shorttitle` option in the header.

**Comment by crsh** [comment]
Hi Peder, thanks for sharing your solution. This has come up before, and I think this should be a [convenient solution](https://github.com/crsh/papaja/issues/130#issuecomment-493003216). Is this what you had in mind?

**Comment by pederisager** [comment]
Yes thats it exactly! Is this included in papaja documentation somewhere? Did i just miss it?

**Comment by crsh** [comment]
I think I forgot to document this. I've added the documentation label to this issue and will keep it open as a reminder to add this to the documentation. Thanks again for reaching out about this.
